cas-server-support-ldap/src/main/java/org/jasig/cas/authentication/LdapAuthenticationHandler.java

```diff
@@ -152,7 +153,7 @@ protected HandlerResult authenticateUsernamePasswordInternal(final UsernamePassw
         final AuthenticationResponse response;
         try {
             logger.debug("Attempting LDAP authentication for {}", upc);
-            final String password = getPasswordEncoder().encode(upc.getPassword());
+            final String password = Strings.nullToEmpty(getPasswordEncoder().encode(upc.getPassword()));
```

Comment:
Very good. So, I think we should take care of all other handlers as well since they may be prone to the issue?

Also, how is that the password ends up being null in the first place?